climada/hazard/tc_tracks.py

```diff
@@ -1119,13 +1119,16 @@ def extent(self):
         """Exact extent of trackset as tuple, no buffer."""
         return self.get_extent(deg_buffer=0.0)
 
-    def plot(self, axis=None, **kwargs):
+    def plot(self, axis=None, legend=True, **kwargs):
```

Comment:
I think it is fine that you added the possibility to turn off the legends, but it is not very clean to make it in an unrelated pull request.

Reply:
You are right, do you want me to set that back? However I guess just this is not worth a pull request...

Reply:
No no, just for the future ;)